message_ix/tests/test_feature_temporal_level.py

```diff
@@ -0,0 +1,568 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
It's great there are detailed comments in these tests! In general:
- When there are inline comments ahead of a specific test function (e.g. `test_season_to_year()`), move them to a docstring inside the function itself.
- For docstrings, our style guide points to PEP 8 which in turn points to [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) (this could be more direct). Using `black` doesn't modify comments at all, however, so these guidelines have to be applied manually:
  - “Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.”
  - “Unless the entire docstring fits on a line, place the closing quotes on a line by themselves.” (and usually without a preceding blank line)

Reply:
Thanks a lot, and sorry for overlooking this. The issues resolved.

Reply:
Looks good, thanks.

In the future, you can put the first line on the same line as the opening triple-quote:
```py
def add_cap_par(scen, years, tec, data={"inv_cost": 0.1, "technical_lifetime": 5}):
    """Add required parameters for representing "capacity" in a model.

    Parameters
    ----------
    scen : Class
        message_ix.Scenario.
    years : list of integers
        Years for adding data.
    tec : string
        Technology name for adding capacity parameters.
    data : Dict, optional
        Dictionary of parameters and their values.
        The default is {"inv_cost": 0.1, "technical_lifetime": 5}.
    """
    ...
```